### ISSUE TITLE
Rope does not disappear if you pick up the crate the rope is attached to

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#62]
```
What steps will reproduce the problem?
1. Target a crate (i tried it with a banner-crate) 
2. Shoot the rope at the crate so it's attached to it
3. Pull yourself on the rope so that your hedgehog picks up the crate

What is the expected output? What do you see instead?

The output should be when you touch the crate and it disappears your rope would 
disappear too, since there is nothing to keep it in place. Instead it keeps 
being mid-air.


What version of the product are you using? On what operating system?

Windows 7 32-bit, 0.9.13.

Please provide any additional information below.

In other games crates cannot be target of the rope, they are "invisible" and 
you can aim only at the land behind them.


```

-----
Original issue reported on code.google.com by `invalida...@gmail.com` on 24 Sep 2010 at 9:12

**Comment by GoogleCodeExporter** [comment]
```
Trivial to make ropes blind to hogs, crates, barrels, whatever...
Just, kind of a subject of disagreement.
There's a forum thread discussing this.

I dunno, I like current behaviour.  Also, if it did change, I'd just want it to 
detach when land is erased, not block roping to hogs and crates and barrels.

But, in any case. Going to rename this enhancement, although it is more like 
"point of discussion" :)

I suggest continuing this in the forum thread - shouldn't be hard to find.

Also, someone else kinda filed a few rope bugs where he mentioned this and 
other cases of attachment where things vanish.  Not gonna bother duping though 
'cause the one where he properly described crate he kinda veered off onto other 
issues.  This one stays on topic.

```

-----
Original comment by `kyberneticist@gmail.com` on 24 Sep 2010 at 9:39
* Changed state: **Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
BTW, Portals have the behaviour you describe.
Just 'cause it'd be darn silly to have a luminous extradimensional something 
attached to a hog. :

```

-----
Original comment by `kyberneticist@gmail.com` on 24 Sep 2010 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
kyberneticist wrote:

"I dunno, I like current behaviour.  Also, if it did change, I'd just want it 
to detach when land is erased, not block roping to hogs and crates and barrels."

---

Yeah, i like this way too. After picking up the crate, however, the rope hangs 
in mid-air, and that's a bug, and was my intention to report. :)

```

-----
Original comment by `invalida...@gmail.com` on 1 Oct 2010 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
I've been waiting for a fix for this weird problem too for a long time myself, 
it's very annoying because it allows an already very overpowered weapon to make 
even more "cheap" moves.
Hope someone can finally fix it.
```

-----
Original comment by `inufa...@gmail.com` on 1 Oct 2010 at 3:03

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `vittorio...@gmail.com` on 31 Oct 2010 at 5:29

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `kyberneticist@gmail.com` on 1 Nov 2010 at 5:37
* Changed state: **New**
* Added labels: **Type-Enhancement**
* Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4378:ab577db125c4 - you can't rope to crates at all anymore.

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Nov 2010 at 6:26
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified (rope disappears upon crate pickup).
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 8:03
* Changed state: **Verified**
